### ISSUE TITLE
Please pin a tracking issue for nightly proc macros

### ISSUE THREAD

**Issue by TheButlah** [rust-lang/rust-analyzer#12893]
First and foremost, thank you for making such a generally wonderful tool! I love rust analyzer :)

However, I have been using r-a for probably almost a month and a half now, maybe more, with proc macros not expanding. This is proving difficult as my codebase really relies on proc macros. Its also giving off a bad look for rust to others looking into the language as we use the nightly compiler.

I read all of #12806 and #12803 but its not clear to me if/when things will be fixed. Is there a way that a path forward with checkboxes can be pinned and updated to track progress, or at least an issue we can subscribe to to know when things are fixed on nightly?

Its also not clear to me if there is a workaround in the meantime - I think I read that there wasn't, but some clarification would be appreciated.

Edit: I saw #12805 but I didn't find it very descriptive of a path forward or clear, and the title does not explicitly mention proc macros.

**Comment by fasterthanlime** [comment]
See https://github.com/rust-lang/rust-analyzer/issues/12803#issuecomment-1198366093

tl;dr next monday, things should start working out-of-the-box for `nightly-2022-07-29` and later.

**Comment by jonas-schievink** [comment]
All you have to do to fix them in the versioned ABIs is to copy the changes from the sysroot ABI to the 1.64 ABI. You can just follow https://github.com/rust-lang/rust-analyzer/issues/12806 for that I think, so closing.
